core/base-service/base-svg-scraping.js

```diff
@@ -12,7 +12,7 @@ const leadingWhitespace = /(?:\r\n\s*|\r\s*|\n\s*)/g
 class BaseSvgScrapingService extends BaseService {
   static valueFromSvgBadge(svg, valueMatcher = defaultValueMatcher) {
     if (typeof svg !== 'string') {
-      throw TypeError('Parameter should be a string')
+      throw new TypeError('Parameter should be a string')
```

Comment:
This was not actually a bug, but `TypeError` is not part of [CAPS_ALLOWED](https://github.com/eslint/eslint/blob/b9aabe34311f6189b87c9d8a1aa40f3513fed773/lib/rules/new-cap.js#L16).